### DISCUSSION TITLE
Trial to reset mouse position

### DISCUSSION THREAD

**Discussion by kburkin** [jspsych/jsPsych#1150] [Q&A] [answered]
Hi Team!

I'm working on a study where participants will click on an image in an odd one out task. I'd like to have a trial in between test trials where the participant has to click on something at the bottom of the screen, so if they're doing the study with a mouse they aren't left with their cursor sitting over an image when the next trial comes up.

I'm thinking a trial with the instructions plugin where there is no text and the navigation buttons align with the bottom of the screen would be good. Ideally, if that were the case, I'd have that apply to my actual instructions pages as well to keep the location of the Next button constant. However I have no idea how to manipulate the position of navigation buttons. Does anyone have any suggestion for how I might do that, or an alternate suggestion for how to make such a trial?

-Kelly

**Comment by jodeleeuw** [comment, marked as answer]
Hi @kburkin ,

I'd suggest using an `html-button-response` trial. You can customize the style of the button to fix it at the bottom of the screen.

```js
{
      type: 'html-button-response',
      stimulus: '',
      choices: ['click here'],
      button_html: '<button style="position:fixed; bottom: 0;" class="jspsych-btn">%choice%</button>'
}
```

**Reply by kburkin** [reply]
Ah yes, thank you that is definitely better! The buttons for this one do wind up sticking to the bottom, however they fall a little to the right of center. Is there any way to adjust that to center them as well?

**Reply by jodeleeuw** [reply]
Here's a small modification to center it. It requires specify the width of the button. You can change that to suit whatever text you put in it.

```js
{
      type: 'html-button-response',
      stimulus: '',
      choices: ['click here'],
      button_html: '<button style="position:fixed; width:100px; left:calc(50vw - 50px); bottom: 0;" class="jspsych-btn">%choice%</button>'
}  
```

**Comment by kburkin** [comment]
Thank you, that works perfectly. I appreciate your help!
